core/src/main/resources/hudson/PluginManager/_table.css

```diff
@@ -8,3 +8,10 @@
     margin: 1em;
     text-align: right;
 }
+.category-label {
```

Comment:
I'd rename this class to `.plugin-manager-category-label`, as `.category-label` is too generic.

Reply:
This CSS is only loaded within the plugin manager (see file path). Still?

Reply:
Then it's not that big of a deal. I still recommend trying to namespace CSS in case we kill / group adjuncts one day (which I really, really want to do)